File: src/2011-Final-Value-of-Variable-After-Performing-Operations/final-value-of-variable-after-performing-operations.py
from typing import List

class Solution:
    def finalValueAfterOperations(self, operations: List[str]) -> int:
        x: int = 0
        for s in operations:
            if '+' in s:
                x += 1
            else:
                x -= 1
        return x

if __name__ == "__main__":
    operations: str = ["--X","X++","X++"]
    solution = Solution()
    
    print(solution.finalValueAfterOperations(operations))

File: src/2011-Final-Value-of-Variable-After-Performing-Operations/final-value-of-variable-after-performing-operations.cpp
#include <iostream>
#include <vector>
#include <string>

class Solution {
public:
    int finalValueAfterOperations(std::vector<std::string>& operations) {
        int x = 0;
        
        for (const std::string& s : operations) {
            if (s.find('+') != std::string::npos) {
                x += 1;
            } else {
                x -= 1;
            }
        }
        return x;
    }
};

int main() {
    std::vector<std::string> operations = {"--X","X++","X++"};

    Solution solution;

    std::cout << solution.finalValueAfterOperations(operations);

    return 0;
}